tests/test_infer.py

```diff
@@ -72,18 +73,20 @@ def test_toms748_scan_bounds_extension(hypotest_args):
         data, model, 3, 5, rtol=1e-8
     )
 
-    assert observed_limit == pytest.approx(observed_limit_default)
+    # FIXME: Remove float cast after TensorFlow support removed
```

Comment:
@kratsg Running the TensorFlow tests is _very_ slow and adds minutes to this run. As we're going to be dropping TensorFlow in the future, I think that we could probably just skip TensorFlow in these tests. Thoughts?